src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java

```diff
@@ -4,9 +4,6 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.foodrem.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.foodrem.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
```

Comment:
Was wondering whether this is supposed to be replace with this

import static seedu.foodrem.commons.core.Messages;

Reply:
Both ways are possible. When it's clear and there's only a small amount of imports, sometimes we just use static imports so we don't have to call the constants by their fully qualified names. I chose to keep the static imports here.

Reply:
I see. Thanks for the explanation!!